website/docs/plugins/submitit_launcher.md

```diff
@@ -83,7 +83,7 @@ For more details, including descriptions for each parameter, check out the [conf
 
 An [example application](https://github.com/facebookresearch/hydra/tree/master/plugins/hydra_submitit_launcher/example) using this launcher is provided in the plugin repository.
 
-Starting the app with `python my_app.py task=1,2,3 -m` will launch 3 executions (you can override the launcher to run locally for testing by adding `hydra/launcher=submitit_local`):
+Starting the app with `python my_app.py task=1,2,3 -m` (where `-m` is a shortcut for `--multirun`) will launch 3 executions (you can override the launcher to run locally for testing by adding `hydra/launcher=submitit_local`):
```

Comment:
just specify --multirun instead.

Reply:
That was my first thought but then I realized that if the doc only showed `--multirun` then I'd still be using that myself and wasting time typing all these extra characters.
What about:
```
Starting the app with `python my_app.py task=1,2,3 --multirun` (`-m` in abbreviated form) ...
```

Reply:
You can't explain everything everywhere.
This is described [here](https://hydra.cc/docs/next/tutorials/basic/running_your_app/multi-run).
Maybe add a link if you think it's better.

Reply:
What about 39ba5d5e?